### PR TITLE
Add eval script for benchmarking multiple datasets 

### PR DESCRIPTION
**Pull request by hqkate**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: tools/eval_benchmark.py
Comment (SamitHuang) [inline tools/eval_benchmark.py:1]: 脚本易与eval.py混淆，需要作调整，放在tools/benchmarking/multi_dataset_eval.py
同时，在.py docstring 或 tools/benchmarking/README.md 中给出usage.

**Review by SamitHuang — COMMENTED** [review]
need update

File: tools/eval_benchmark.py
Comment (hqkate) [reply tools/eval_benchmark.py:1]: 已修改

File: tools/benchmarking/multi_dataset_eval.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+            if not reload_data:
+                net_evaluator = Evaluator(
+                    network,
+                    loader_eval,
+                    loss_func=None,
+                    postprocessor=postprocessor,
+                    metrics=[metric],
+                    num_columns_to_net=dataset_config.get('num_columns_to_net', 1),
+                    num_columns_of_labels=dataset_config.get('num_columns_of_labels', None),
```
Comment (SamitHuang) [inline tools/benchmarking/multi_dataset_eval.py:127]: set num_epochs=1

File: tools/benchmarking/multi_dataset_eval.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+                    )
+                reload_data = True
+            
+            else:
+                net_evaluator.reload(
+                    loader_eval,
```
Comment (SamitHuang) [inline tools/benchmarking/multi_dataset_eval.py:133]: set num_epochs=1

Comment (hqkate) [reply tools/benchmarking/multi_dataset_eval.py:134]: Updated

**Review by HaoyangLee — APPROVED** [review]

File: tools/benchmarking/multi_dataset_eval.py
Comment (SamitHuang) [inline tools/benchmarking/multi_dataset_eval.py:1]: 这个评估脚本在crnn中的评估使用，请加到configs/rec/crnn的readme中。

File: tools/benchmarking/multi_dataset_eval.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    ```
+
+    You can then get the performance of each individual dataset as well as the average score under evaluation/.
+'''
+
+import sys
+import os
+import copy
+__dir__ = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.abspath(os.path.join(__dir__, "..")))
```
Comment (zhtmike) [inline tools/benchmarking/multi_dataset_eval.py:45]: 因为是在主目录第二层， 所以应该是 `sys.path.insert(0, os.path.abspath(os.path.join(__dir__, "../..")))`

Comment (hqkate) [reply tools/benchmarking/multi_dataset_eval.py]: thanks, fixed

File: tools/benchmarking/multi_dataset_eval.py
Comment (hqkate) [reply tools/benchmarking/multi_dataset_eval.py:1]: 好的，会在 #191 中添加这部分的说明

**Review by zhtmike — APPROVED** [review]

**Pull request merged**